paddlenlp/seq2vec/encoder.py

```diff
@@ -24,46 +24,51 @@
 
 
 class BoWEncoder(nn.Layer):
-    """
+    r"""
     A `BoWEncoder` takes as input a sequence of vectors and returns a
     single vector, which simply sums the embeddings of a sequence across the time dimension. 
     The input to this module is of shape `(batch_size, num_tokens, emb_dim)`, 
     and the output is of shape `(batch_size, emb_dim)`.
 
     Args:
-        emb_dim(int): It is the input dimension to the encoder.
+        emb_dim(`int`): 
+            It is the input dimension to the encoder.
     """
 
     def __init__(self, emb_dim):
         super().__init__()
         self._emb_dim = emb_dim
 
     def get_input_dim(self):
-        """
+        r"""
         Returns the dimension of the vector input for each element in the sequence input
         to a `BoWEncoder`. This is not the shape of the input tensor, but the
         last element of that shape.
         """
         return self._emb_dim
 
     def get_output_dim(self):
-        """
+        r"""
         Returns the dimension of the final vector output by this `BoWEncoder`.  This is not
         the shape of the returned tensor, but the last element of that shape.
         """
         return self._emb_dim
 
     def forward(self, inputs, mask=None):
-        """
+        r"""
         It simply sums the embeddings of a sequence across the time dimension.
 
         Args:
-            inputs (paddle.Tensor): Shape as `(batch_size, num_tokens, emb_dim)`
-            mask (obj: `paddle.Tensor`, optional, defaults to `None`): Shape same as `inputs`. Its each elements identify whether is padding token or not. 
+            inputs (`Tensor`):
+                Shape as `(batch_size, num_tokens, emb_dim)`
+            mask (`Tensor`, optional): 
+                Shape same as `inputs`. Its each elements identify whether is padding token or not. 
```

Comment:
Its each elements identify whether is padding token or not. 
->
Each entry of the `mask` tensor is a boolean value identifying whether the corresponding token in the `inputs` is padding or not.